From 2b53cdb61283910e4b4fe2d45e85b2276cc15784 Mon Sep 17 00:00:00 2001
From: Marko Lindqvist <cazfi74@gmail.com>
Date: Fri, 6 Dec 2024 01:23:01 +0200
Subject: [PATCH 1/3] Mark flathub as the distributor

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
---
 fc_version | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fc_version b/fc_version
index 46f044d18e..f9d2fb4fc1 100755
--- a/fc_version
+++ b/fc_version
@@ -76,7 +76,7 @@ DEFAULT_FOLLOW_TAG=S3_2
 #     so would break network capability of supposedly "compatible" releases.
 #
 NETWORK_CAPSTRING="+Freeciv-3.2-network"
-FREECIV_DISTRIBUTOR=""
+FREECIV_DISTRIBUTOR="flathub"
 
 if test "x$FREECIV_LABEL_FORCE" != "x" ; then
   VERSION_LABEL=$(echo $FREECIV_LABEL_FORCE | sed "s/<base>/$VERSION_LABEL/g")
-- 
2.45.2

