From cdd6d6bcf3c5a85bd548b06bf490c222baa9d47e Mon Sep 17 00:00:00 2001
From: Marko Lindqvist <cazfi74@gmail.com>
Date: Thu, 10 Jul 2025 11:09:38 +0300
Subject: [PATCH] Mark flathub as the distributor

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
---
 fc_version | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fc_version b/fc_version
index f9621c56ae..743956bfcb 100755
--- a/fc_version
+++ b/fc_version
@@ -81,7 +81,7 @@ NETWORK_CAPSTRING="+Freeciv-3.2-network ownernull16 unignoresync"
 # patch also this field to contain your identification.
 #
 # See doc/README.packaging chapter "Compatibility of modified versions"
-FREECIV_DISTRIBUTOR=""
+FREECIV_DISTRIBUTOR="flathub"
 
 if test "${FREECIV_LABEL_FORCE}" != "" ; then
   VERSION_LABEL="$(echo "${FREECIV_LABEL_FORCE}" | sed "s/<base>/${VERSION_LABEL}/g")"
-- 
2.47.2

